Simulator/Recorders/XmlRecorder.h

```diff
@@ -69,60 +84,91 @@ class XmlRecorder : public Recorder {
       resultFileName_ = fileName;
    }
 
-   // Accessor method for neuronName_ (only included during unit tests)
+   // Accessor method for variable name (only included during unit tests)
    // @param numIndex   The index number in the variable list.
    std::string getVariableName(int numIndex) const
    {
       return variableTable_[numIndex].variableName_;
    }
```

Comment:
If possible pass string and other object as a reference (or const reference) from getter methods.